var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -412,12 +424,13 @@ def edit(self, spec, prefix):
             ])
 
         if '+elpa' in spec:
+            dso_suffix = 'a' if '%aocc' in spec else 'so'
             elpa = spec['elpa']
             elpa_suffix = '_openmp' if '+openmp' in elpa else ''
             elpa_incdir = elpa.headers.directories[0]
 
             fcflags += ['-I{0}'.format(os.path.join(elpa_incdir, 'modules'))]
-            libs.append(os.path.join(elpa.libs.directories[0],
+            libs.append(os.path.join(elpa.prefix.lib,
```

Comment:
We recommend use of Spack's `join_path` instead of `os.path.join`.  I'm seeing only one use of the former in this package.  Consistency would be nice. :)

Reply:
there were close to 20 occasions where `os.path.join` is used in the package. addressed all of them.